timedColorChange.js

```diff
@@ -13,17 +14,19 @@ function getRandomColor() {
 function setRandomColor() {
     var background = document.getElementsByTagName( 'main' )[0];
     var newColor = getRandomColor();
-
+    // checks if generated color has already been used
     if( used.includes( newColor ) ) {
```

Comment:
@veem-abba ah, good catch! I've updated to use a while loop that resets newColor until a color that hasn't been used is generated.